src/ExifTool.ts

```diff
@@ -5,19 +5,12 @@ import { TagsTask } from "./TagsTask"
 import { VersionTask } from "./VersionTask"
 import { BatchCluster } from "batch-cluster"
 import * as _child_process from "child_process"
-import * as _fs from "fs"
 import * as _os from "os"
-import * as _process from "process"
 
 export { Tags } from "./Tags"
 export { ExifDate, ExifTime, ExifDateTime, ExifTimeZoneOffset } from "./DateTime"
 
-const isWin32 = _process.platform === "win32"
-const exiftoolPath = require(`exiftool-vendored.${isWin32 ? "exe" : "pl"}`)
-
-if (!_fs.existsSync(exiftoolPath)) {
-  throw new Error(`Vendored ExifTool does not exist at ${exiftoolPath}`)
-}
+const exiftoolPath = _child_process.execSync("npm run -s exiftool-path").toString()
```

Comment:
I really don't want to sync-exec `npm` in the middle of the require flow. I'll rework this before it goes into master.